### PR TITLE
feat: add CI for windows setup scripts

### PR DESCRIPTION
**Pull request by dtopuzov**
New:
- Add AppVeyor ci for windows setup

Fixed:
- Silent mode is now working

Refactor:
- Install OpenJDK 8 instead of Oracle JDK 8
- Combine all the setup in single script

Notes:
- After this PR we do not longer need "https://www.nativescript.org/setup/win-common" url.
- This PR incorporate all the changes from https://github.com/NativeScript/setup-scripts/pull/21

### CODE REVIEW

File: native-script.ps1 [hunk 1]
```diff
@@ -167,8 +179,51 @@ echo y | cmd /c "$androidExecutable" "extras;google;m2repository"
...
+if ($installEmulatorAnswer -eq 'y') {
+	Write-Host -ForegroundColor DarkYellow "Setting up Android SDK system-images;android-28;google_apis;x86..."
+	echo y | cmd /c "$androidExecutable" "system-images;android-28;google_apis;x86"
+
+	echo y | cmd /c "$androidExecutable" "extras;intel;Hardware_Accelerated_Execution_Manager"
+	$haxmSilentInstaller = [io.path]::combine($env:ANDROID_HOME, "extras", "intel", "Hardware_Accelerated_Execution_Manager", "silent_install.bat")
+	cmd /c "$haxmSilentInstaller"
+
+	if ($LASTEXITCODE -ne 0) {
+		Write-Host -ForegroundColor Yellow "WARNING: Failed to install HAXM in silent mode. Starting interactive mode."
```
[the remark this answers is not in the document]
Comment (rosen-vladimirov) [reply native-script.ps1]: my mistake - we are running silent install

File: native-script.ps1 [hunk 2]
```diff
@@ -167,8 +179,51 @@ echo y | cmd /c "$androidExecutable" "extras;google;m2repository"
 Write-Host -ForegroundColor DarkYellow "FINISHED setting up Android SDK."
 
 # Setup Default Emulator
-iex ((new-object net.webclient).DownloadString($scriptCommonUrl))
-Create-AVD
+$androidExecutable = [io.path]::combine($env:ANDROID_HOME, "tools", "bin", "sdkmanager")
+$avdManagerExecutable = [io.path]::combine($env:ANDROID_HOME, "tools", "bin", "avdmanager")
+
+# Emulator will be not installed in silent mode since Intel HAXM fails to install in silent mode.
```
[the remark this answers is not in the document]
Comment (dtopuzov) [reply native-script.ps1]: Fixed.

**Comment by etabakov** [comment]
@cla-bot check

**Comment by dtopuzov** [comment]
test

**Comment by dtopuzov** [comment]
run ci

**Pull request merged**